### PR TITLE
Format standards for Add and Edit

### LINKED ISSUE #184: Update UG for Add and Edit

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #184 
This PR will:
* Update `Add` and `Edit` commands according to the standardized styles.
* Add in related **Tips** and **Constraints**
* Small change which is different from the style: the **Constraints** box of `Add` command has the header of `##### Constraints` instead of being inline with the `<box>` tag
> Reason: so that it can be referred to from the `Edit` command **Constraints**

### CODE REVIEW

**Review by teamotitan — APPROVED** [review]
Looks good to me!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -89,57 +92,65 @@ Adds a student to the address book.
...
 **Format:** `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
 
 **Examples:**
 
-`edit 1 p/87438808 e/alexyeoh100@example.com` </br> Edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
+* `edit 1 p/87438808 e/alexyeoh100@example.com` edits the phone number and email address of the 1st student to be `87438808` and `alexyeoh100@examnple.com` respectively.
 
-`edit 2 paid/1200.00 owed/0` </br> Edits the paid amount of the 2nd student to be `$1200.00` and edits the owed amount to be `$0.00`.
+* `edit 2 paid/1200.00 owed/0` edits the paid amount of the 2nd student to be `$1200.00` and edits the owed amount to be `$0.00`.
 ![editResult.png](images/editResult.png)
```
Comment (teamotitan) [inline docs/UserGuide.md:135]: For this image, do you want to consider following the same as add command, like the Output:

**Review by teamotitan — COMMENTED** [review]
Looks good to me, just wondering about placement of 1 image

File: docs/UserGuide.md (hunk 1 above)
Comment (DennieDan) [reply docs/UserGuide.md:135]: Thanks for the suggestion, I will change

**Review by teamotitan — APPROVED** [review]
Looks good to me!

**Review by Feng1231 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -70,16 +73,16 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
-Format: `help`
+**Format**: `help`
 
 
 ### Adding a student: `add`
 
-Adds a student to the address book.
+Adds a student to UG Teach.
 
 **Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID] [owed/OWED]`
```
Comment (Feng1231) [inline docs/UserGuide.md:87]: PAID and OWED to be updated

File: docs/UserGuide.md [hunk 3]
```diff
@@ -89,57 +92,68 @@ Adds a student to the address book.
...
+* **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
+* **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
+* **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
+* **RATE**, **PAID** and **OWED** must be at least 0 with at most 2 decimal places.
       <i>Example: </i> `12.00`, `0.0` or `7`
-6. **SUBJECT** should only be:
+* **SUBJECT** should only be
 `Economics`  `Literature`  `Music`  `Biology`  `Chemistry`  `Science`  
-`English`  `Chinese`  `Malay` `Tamil`  `Mathematics`  `History`  `Geography`  `Physics`  `GP`
+`English`  `Chinese`  `Malay` `Tamil`  `Mathematics`  `History`  `Geography`  `Physics` and `GP`
```
Comment (Feng1231) [inline docs/UserGuide.md:107]: instead of and, would it be better if we use **or**?

Subject should be A, B,..., OR Z

**Review by Feng1231 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (Feng1231) [reply docs/UserGuide.md:87]: same for in the command summary

File: docs/UserGuide.md [hunk 4]
```diff
@@ -70,43 +73,45 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
+<box type="important">
 
-1. **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
-2. **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
-3. **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
-4. **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
-5. **RATE**, **PAID** and **OWED** must be at least 0 with at most 2 decimal places.
-      <i>Example: </i> `12.00`, `0.0` or `7`
-6. **SUBJECT** should only be:
+##### Constraints
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:97]: Could consider using same format as line 111.

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM. Just minor consideration

File: docs/UserGuide.md (hunk 4 above)
Comment (DennieDan) [reply docs/UserGuide.md:97]: Hmmm, this is because I want to refer to it in the `edit` command

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**